irc/channel.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+export type ChannelType =
```

Comment:
Channel Mode? I forget what all these things mean - it would be nice to have a small one line comment describing them